tests/onnxruntime/test_modeling.py

```diff
@@ -108,6 +108,7 @@
     "flaubert": "hf-internal-testing/tiny-random-flaubert",
     "gpt2": "hf-internal-testing/tiny-random-gpt2",
     "gpt_neo": "hf-internal-testing/tiny-random-GPTNeoModel",
+    "gpt_neo_x": "hf-internal-testing/tiny-random-GPTNeoXForCausalLM",
```

Comment:
Could you add `gpt_neo_x` in the `ORTModelForCausalLMIntegrationTest`'s `SUPPORTED_ARCHITECTURES`? This allows to test it using the ONNX Runtime integration.

Reply:
Added!